packages/governance/test/swingsetTests/contractGovernor/test-governor.js

```diff
@@ -98,11 +98,49 @@ const expectedcontractGovernorStartLog = [
   'updated to "[299792458n]"',
   'MalleableNumber was changed to "[602214090000000000000000n]"',
   'current value of MalleableNumber is 299792458',
+  'Electorate was changed to {"brand":"[Alleged: Zoe Invitation brand]","value":[{"description":"questionPoser","expiry":"[undefined]","fee":"[undefined]","handle":"[Alleged: InvitationHandle]","installation":"[Alleged: Installation]","instance":"[Alleged: InstanceHandle]","zoeTimeAuthority":"[undefined]"}]}',
   'MalleableNumber was changed to "[299792458n]"',
   'Voter Alice validated all the things',
 ];
 
-test('zoe - contract governance', async t => {
+test.serial('contract governance', async t => {
```

Comment:
Why does this need to be `serial`? worth a comment?

Reply:
If `serial` was a temp thing and you're changing it back, consider also changing the title from a noun phrase ("contract governance") to a test assertion ("contract governance solves all the world's problems" :).